docs/projects/fractions/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Fractions in Every Language
+layout: default
+date: 2019-10-10
+last-modified: 2019-10-10
+feature-image:
+tags: [fractions]
+authors:
+---
+
+languages like python have built-in utilities or functions for working with fractions.
+Many of these fractions functions follow a similar pattern across programming languages:
+Takes a numerator and a denomenator as an attribute.
+Perform basic arithmatic and relational operations with operator overloading.
+
+## Requirements
+
+In general, a fractions library should perform the following:
+
+1. Perform arithmatic operation like multiplications, addition etc.
+2. Give output for relational operations like >=, >, == etc.
+
+More specifically, begin with creating object instance of fraction class with two attributes:
+numerator and denomenator.Using operator overloading feature of langauge implement basic arithmatic
+and relational operaions.
+
+## Testing
+
+Verify that the actual output matches the expected output.
+
+- Fill as needed
```

Comment:
I'll fill in this part (when I remember)! 😄

Reply:
thanks